pkgs/by-name/gi/gitcs/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "gitcs";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "knbr13";
+    repo = "gitcs";
+    rev = "v${version}";
+    hash = "sha256-IyhVVRTKftZIzqMH5pBUMLPIk8bk0rVAxPKD6bABP68=";
+  };
+
+  vendorHash = "sha256-8yzPdVljnODOeI5yWh19BHsF4Pa9BWc49IwenMCVGZo=";
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  meta = with lib; {
+    description = "Scan local git repositories and generate a visual contributions graph.";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

must not end in period

Reply:
Changed it